lib/index.ts

```diff
@@ -67,10 +53,15 @@ export const useStoryblok = (
   return story;
 };
 
-export const useStoryblokState = <T = void>(
-  initialStory: ISbStoryData<T> | null = null,
-  bridgeOptions: StoryblokBridgeConfigV2 = {}
-): ISbStoryData<T> | null => {
+type tUseStoryblokState = <T = void>(
+  initialStory: ISbStoryData<T> | null,
+  bridgeOptions: StoryblokBridgeConfigV2
+) => ISbStoryData<T> | null;
+
+export const useStoryblokState: tUseStoryblokState = (
```

Comment:
needs to be removed as @arorachakit catched! already implemented in `common/client.ts`